docs/diagnostics/PX1035.md

```diff
@@ -0,0 +1,94 @@
+# PX1035
+This document describes the PX1035 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                 | Type  | Code Fix    | 
+| ------ | ------------------------------------------------- | ----- | ----------- | 
+| PX1035 | The DAC has multiple primary key declarations | Warning (ISV Level 3: Informational) | Available | 
+
+## Diagnostic Description
+A DAC cannot have multiple primary key declarations. The warning is shown only for DACs that have the `PXCacheName` or `PXPrimaryGraph` attribute.
+The diagnostic does not check abstract DACs and DACs which have only unbound DAC properties (so-called "virtual" DACs used for custom popups and filters for inquiry forms).
```

Comment:
@dropsonic, do we use term "virtual" for DACs in official documentation? Is it correct to use it or the term "fully-unbound" DAC?

Reply:
I think "unbound" is more correct; the best way is to add a proper clarification ("DAC that is not bound to any DB table and exists only in-memory").